### DISCUSSION TITLE
Ring Intercom lock + HomeKit Bridge

### DISCUSSION THREAD

**Discussion by simps100** [tsightler/ring-mqtt#687] [General Discussion]
Hey

Has anyone who has the ring intercom managed to get the lock to show in Apple home?

I've setup the bridge and have setup my front entrance lock as an included entity along with my Ring alarm but the lock never appears in Home (The alarm shows fine) - not sure if this just isn't supported or if it's a problem / limitation with the HomeKit bridge add on.

Thank you!
Marc

**Comment by simps100** [comment]
Worked it out - have to create it as an accessory rather than a bridge just incase anyone runs into the same and ends up here! :)

**Reply by tsightler** [reply]
Thanks for sharing.  I almost suggested this since the documentation mentioned something about this, but then I realized I had no idea what I was talking about since I don't use Homekit.
